src/main/java/jp/co/future/uroborosql/converter/SingleColumnResultSetConverter.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2017-present, Future Corporation
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package jp.co.future.uroborosql.converter;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.Month;
+import java.time.MonthDay;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.Year;
+import java.time.YearMonth;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jp.co.future.uroborosql.exception.UroborosqlRuntimeException;
+import jp.co.future.uroborosql.mapping.JavaType;
+import jp.co.future.uroborosql.mapping.mapper.PropertyMapperManager;
+import jp.co.future.uroborosql.utils.CaseFormat;
+import jp.co.future.uroborosql.utils.StringUtils;
+
+/**
+ * 検索結果の1行をプリミティブ型（のラッパークラス）に変換する変換器
+ *
+ * @param <E> プリミティブ型のラッパークラス、またはString型
+ * @author H.Sugimoto
+ *
+ */
+public class SingleColumnResultSetConverter<E> implements ResultSetConverter<E> {
+	/** ロガー */
+	private static final Logger LOG = LoggerFactory.getLogger(SingleColumnResultSetConverter.class);
+
+	private final String col;
+	private final JavaType javaType;
+	private final PropertyMapperManager mapperManager;
+	private int columnPosition;
+
+	/**
+	 * 受付可能な型かどうかを判定する.
+	 *
+	 * @param type 判定対象の型
+	 * @return 受付可能な場合<code>true</code> を返す.
+	 */
+	public static final boolean accept(Class<?> type) {
```

Comment:
@HidekiSugimoto189 
`PropertyMapperManager` also supports the following types that do not exist in the branch of this method, so shouldn't it be accepted by this method?

- Domain
- Enum
- java.sql.Timestamp
- java.sql.Time
- java.sql.Date
- java.sql.Array
- java.sql.Blob
- java.sql.Clob
- java.sql.NClob
- java.sql.Ref
- java.sql.SQLXML
- Optional
- OptionalDouble
- OptionalInt
- OptionalLong

Reply:
@shout-star Thanks for the review.   
I've added the classes you pointed out as targets. I've also added the test cases.